dhall-csv/src/Dhall/Csv.hs

```diff
@@ -62,11 +62,16 @@ dhallToCsv e0 = listConvert $ Core.normalize e0
         return $ Data.Csv.toNamedRecord $ Dhall.Map.toMap a'
     recordConvert e = Left $ Unsupported e
     fieldConvert :: Expr Void Void -> Either CompileError Data.Csv.Field
+    fieldConvert (Core.BoolLit True) = return $ toField ("true" :: Text)
+    fieldConvert (Core.BoolLit False) = return $ toField ("false" :: Text)
     fieldConvert (Core.NaturalLit a) = return $ toField a
     fieldConvert (Core.IntegerLit a) = return $ toField a
     fieldConvert (Core.DoubleLit (DhallDouble a)) = return $ toField a
     fieldConvert (Core.TextLit (Core.Chunks [] a)) = return $ toField a
     fieldConvert (Core.App (Core.Field (Core.Union _) _) a) = fieldConvert a
+    fieldConvert (Core.Field (Core.Union _) (Core.FieldSelection _ k _)) = return $ toField k
+    fieldConvert (Core.Some e) = fieldConvert e
+    fieldConvert (Core.App Core.None _) = return $ toField ("null" :: Text)
```

Comment:
My impression was that missing values are often represented as the empty string (`,,`). Is `null` widely used too?

Reply:
I actually didn't give it much thought haha now that you say it is way better the empty string than null 😅 I'll change it right away